docs/package.json

```diff
@@ -23,19 +24,35 @@
       "current node"
     ]
   },
+  "resolutions": {
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "eslint": "^8.56.0"
+  },
   "dependencies": {
+    "@babel/cli": "^7.25.9",
+    "@babel/core": "^7.21.4",
+    "@babel/eslint-parser": "^7.21.3",
+    "@babel/eslint-plugin": "^7.19.1",
+    "@babel/generator": "^7.21.4",
     "@babel/plugin-syntax-jsx": "^7.18.6",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.4",
+    "@babel/runtime": "7.26.7",
     "@babel/standalone": "^7.18.9",
+    "@babel/types": "^7.21.4",
     "@csstools/convert-colors": "^2.0.0",
     "@icons-pack/react-simple-icons": "^8.0.1",
     "@jsdevtools/rehype-url-inspector": "^2.0.2",
-    "@kiwicom/babel-plugin-orbit-components": "*",
-    "@kiwicom/orbit-components": "*",
-    "@kiwicom/orbit-design-tokens": "*",
-    "@kiwicom/orbit-tailwind-preset": "*",
-    "@kiwicom/orbit-themer": "*",
+    "@kiwicom/babel-plugin-orbit-components": "link:../packages/babel-plugin-orbit-components",
+    "@kiwicom/orbit-components": "link:../packages/orbit-components",
+    "@kiwicom/orbit-design-tokens": "link:../packages/orbit-design-tokens",
+    "@kiwicom/orbit-tailwind-preset": "link:../packages/orbit-tailwind-preset",
+    "@kiwicom/orbit-themer": "link:../packages/orbit-themer",
```

Comment:
TIL that this works for deployments 🤔 did you do any setup on Cloudflare? Or the link just works seamlessly?